aiofcm/connection.py

```diff
@@ -247,6 +288,19 @@ async def send_message(self, message: Message) -> MessageResponse:
                 logger.error('Could not send message %s: %s',
                              message.message_id, e)
 
+    async def maintain_min_connections_open(self):
+        while True:
+            missing_connections = max(0, self.min_connections - len(self.connections))
+            if missing_connections > 0:
+                logger.debug('Creating %d missing connections' % missing_connections)
```

Comment:
Don't use string formatting in logger messages in this way because some things like Sentry would consider this as different events. So, replace percent with comma.

Reply:
Fixed